### ISSUE TITLE
deno bundle on a file with unstable apis gives wrong error

### ISSUE THREAD

**Issue by lucacasonato** [denoland/deno#5646]
```bash
$ deno bundle https://gist.githubusercontent.com/lucacasonato/05d9cca0f1405e84c0b614c0c47fef79/raw/e09da8367f19b1082dfd3f9d6e4c77a3ec682921/unstable.ts
error: missing field `bundleOutput` at line 1 column 371
```

It works when the `--unstable` flag is provided

**Comment by bartlomieju** [comment]
In 1.0.1:
```
▶ deno bundle https://gist.githubusercontent.com/lucacasonato/05d9cca0f1405e84c0b614c0c47fef79/raw/e09da8367f19b1082dfd3f9d6e4c77a3ec682921/unstable.ts
Bundling https://gist.githubusercontent.com/lucacasonato/05d9cca0f1405e84c0b614c0c47fef79/raw/e09da8367f19b1082dfd3f9d6e4c77a3ec682921/unstable.ts
Download https://gist.githubusercontent.com/lucacasonato/05d9cca0f1405e84c0b614c0c47fef79/raw/e09da8367f19b1082dfd3f9d6e4c77a3ec682921/unstable.ts
error: TS2339 [ERROR]: Property 'utime' does not exist on type 'typeof Deno'.
Deno.utime();
     ~~~~~
    at https://gist.githubusercontent.com/lucacasonato/05d9cca0f1405e84c0b614c0c47fef79/raw/e09da8367f19b1082dfd3f9d6e4c77a3ec682921/unstable.ts:1:6
```

This should be fixed as part of #5503

**Comment by actual-size** [comment]
@bartlomieju Looks like this has been fixed in the latest release
